<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1214</width>
    <height>659</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>我的通讯录</string>
  </property>
  <property name="iconSize">
   <size>
    <width>60</width>
    <height>60</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="Table_Student">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>1041</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(255, 196, 190);
selection-color: rgb(0, 0, 0);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <row/>
    <column>
     <property name="text">
      <string>学号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>性别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>年龄</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>手机号码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>家庭地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>QQ号码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>专业</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>161</width>
      <height>431</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="allColumnsShowFocus">
     <bool>false</bool>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add"/>
   <addaction name="separator"/>
   <addaction name="action_delete"/>
   <addaction name="separator"/>
   <addaction name="action_edit"/>
   <addaction name="separator"/>
   <addaction name="action_search"/>
   <addaction name="separator"/>
   <addaction name="action_backup"/>
   <addaction name="separator"/>
   <addaction name="action_recovery"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_add">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/picture/ADD.png</normaloff>:/res/picture/ADD.png</iconset>
   </property>
   <property name="text">
    <string> 添加</string>
   </property>
   <property name="iconText">
    <string notr="true">添加</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/picture/DELETE.png</normaloff>:/res/picture/DELETE.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/picture/EDIT.png</normaloff>:/res/picture/EDIT.png</iconset>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </action>
  <action name="action_search">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/picture/search.png</normaloff>:/res/picture/search.png</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="action_backup">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/picture/backup.png</normaloff>:/res/picture/backup.png</iconset>
   </property>
   <property name="text">
    <string>备份</string>
   </property>
  </action>
  <action name="action_recovery">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/picture/recovery.png</normaloff>:/res/picture/recovery.png</iconset>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
